contracts/main/managers/GetPositions.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
We might be able to remove contract GetPositions from the codebase to save audit fee if neither Graph nor Liq. bot uses this.

Reply:
Need to ask Tarun-ji and Vitalii B(to see it's used in front-end)